app/services/github.rb

```diff
@@ -86,4 +84,12 @@ def parse_org_repo(url)
   def set_date_format(*dates)
     dates.map { |date| date.strftime("%Y-%m-%d") }
   end
+
+  private
+
+    def client
+      @client ||= Octokit::Client.new(client_id: ENV['GITHUB_KEY'],
+                                      client_secret: ENV['GITHUB_SECRET'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.